chainer/functions/connection/convolution_2d.py

```diff
@@ -308,7 +308,7 @@ def convolution_2d(x, W, b=None, stride=1, pad=0, use_cudnn=True,
     If the bias vector is given, then it is added to all spatial locations of
     the output of convolution.
 
-    .. seealso:: :class:`Convolution2D`
+    .. seealso:: :class:`chainer.links.Convolution2D`
```

Comment:
Could you change `~chainer.links.Convolution2D` (i.e. add tilde at the head of the class name). It hides the namespace `chainer.links` from the display name.